otter/integration/lib/test_cloud_load_balancer.py

```diff
@@ -235,6 +238,120 @@ def add(clb, clock):
         self.assert_mutate_function_does_not_retry_if_not_pending_update(
             add, main_treq_args)
 
+    def get_fake_treq_for_delete(self, get_json_body, get_response=None,
+                                 del_response=None):
+        """
+        Return a CLB for use with deleting a CLB - this is different than
+        the one returned by `get_clb` because it requires stubbing out two
+        treq requests.
+        """
+        get_response = get_response or Response(200)
+        del_response = del_response or Response(202)
+
+        class FakeTreq(object):
+            def delete(cls, _url, *args, **kwargs):
+                # args and kwargs are the same as the get ones
+                self.assertEqual(args, ())
+                self.assertEqual(kwargs, self.expected_kwargs)
+                self.assertEqual(_url, 'clburl/loadbalancers/{0}'.format(
+                    self.clb_id))
+                return succeed(del_response)
+
+            def get(cls, _url, *args, **kwargs):
+                cls.delete(_url, *args, **kwargs)
+                return succeed(get_response)
+
+            def content(cls, resp):
+                if get_response == Response(200):
+                    self.assertEqual(resp, del_response)
+                else:
+                    self.assertIn(resp, (del_response, get_response))
```

Comment:
Non-blocking: I couldn't understand this `if`. Is it possible to document the intent of `get_response` and `del_response` parameters?